Realm/Tests/MigrationTests.mm

```diff
@@ -200,6 +200,22 @@ - (void)assertNoMigrationRequiredForChangeFrom:(NSArray *)from to:(NSArray *)to
     RLMAssertRealmSchemaMatchesTable(self, [RLMRealm realmWithConfiguration:config error:nil]);
 }
 
+- (void)assertDeletingRealmFilesForChangeFrom:(NSArray *)from to:(NSArray *)to {
```

Comment:
Oh yes, I was going to add two or three more tests. For now, I kept the logic there.